package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "build": "babel src -d lib",
-    "test": "mocha --compilers js:babel-register ./tests/stringIndices_test.js"
+    "test": "mocha --compilers js:babel-register ./tests/disemvowel_test.js"
```

Comment:
you should leave this as `mocha --compilers js:babel-register ./tests/` and just temporarily add `.only` to either  a `describe` or `it` in your mocha tests to run a subset of your tests